internal/controllers/topology/cluster/cluster_controller_test.go

```diff
@@ -734,22 +778,38 @@ func setupTestEnvForIntegrationTests(ns *corev1.Namespace) (func() error, error)
 		WithInfrastructureTemplate(infrastructureMachineTemplate2).
 		WithBootstrapTemplate(bootstrapTemplate).
 		Build()
+	machinePoolClass1 := builder.MachinePoolClass(workerClassName1).
```

Comment:
Can we append a suffix `-mp` for machine pools where we use the workerClassName's and maybe `-md` for machinedeployments?